server/commitment_type.go

```diff
@@ -0,0 +1,107 @@
+package server
+
+import (
+	"encoding/hex"
+	"fmt"
+	"net/http"
+
+	"github.com/Layr-Labs/eigenda-proxy/commitments"
+)
+
+type CommitmentType uint8
+
+const (
+	OptimismCommitmentType CommitmentType = iota
```

Comment:
Given this a commitment package you could just do Optimism since when imported would be referenced like commitment.Optimism

Reply:
Good idea, will do